init/src/synthpop.proto

```diff
@@ -0,0 +1,110 @@
+syntax = "proto3";
+package synthpop;
+
+message Population {
+    repeated Household households = 1;
+    repeated Person people = 2;
+
+    // This is filled out per stringified Activity. The exception is for
+    // `Activity::Home`; see `households` for that.
+    map<string, VenueList> venues_per_activity = 3;
+
+    map<string, InfoPerMSOA> info_per_msoa = 4;
+}
+
+message Household {
+    uint64 id = 1;
+    string msoa = 2;
+    /// An ID from the original data, kept around for debugging
+    int64 orig_hid = 3;
+    // These IDs index into `Population::people`
+    // TODO These're redundant; worth putting here?
+    repeated uint64 members = 4;
+}
+
+message VenueList {
+    repeated Venue venues = 1;
+}
+
+message InfoPerMSOA {
+    repeated Point shape = 1;
+    uint64 population = 2;
+    // All building centroids within this MSOA. May be empty.
+    repeated Point buildings = 3;
+}
+
+// In WGS84
+message Point {
+    float longitude = 1;
+    float latitude = 2;
+}
+
+message Person {
+    uint64 id = 1;
+    uint64 household = 2;
+    // This is the centroid of the household's MSOA. It's redundant to store it per person, but
+    // very convenient.
+    Point location = 3;
+    // An ID from the original data, kept around for debugging
+    int64 orig_pid = 4;
+    // The Standard Industry Classification for where this person works. 0 means none.
+    uint64 sic1d07 = 5;
+
+    uint32 age_years = 6;
```

Comment:
The SPC output is just meant to represent a population at a single point of time. If a downstream model (like ASPICS) wanted to model changes over time, this would be a possible approach.

Relatedly, ASPICS uses lockdown multipliers that have real dates associated. That data isn't plumbed through here yet. If and when we do, we'll have to be careful to represent dates properly

Reply:
and this single point of time has to be 2020 as a matter of fact. It would be great if someone wants to dig into the problem of growing the population every year. SPENSER is supposed to be able to do that already.